src/character.h

```diff
@@ -2313,6 +2313,7 @@ class Character : public Creature, public visitable
         /** Calculates the total fun bonus relative to this character's traits and chapter progress */
         bool fun_to_read( const item &book ) const;
         int book_fun_for( const item &book, const Character &p ) const;
+        mutable std::map<itype_id, int> book_chapters;
```

Comment:
why is this marked as `mutable`? I think it would be better to change `item::mark_chapter_as_read` to not accept `const Character&` instead?

Separately, please add a short comment noting that it represents the amount of chapters *remaining* in a given book, (and not the amount *read* as I originally assumed)